CourseSketchProjects/CourseSketchLoginServer/src/main/java/coursesketch/database/DatabaseClient.java

```diff
@@ -232,14 +232,13 @@ private static String getUserInfo(final DBObject cursor, final boolean loginAsDe
      *            The email of the user.
      * @param isInstructor
      *            If the default account is an instructor
-     * @throws GeneralSecurityException
-     *             Thrown if there are problems creating the hash for the
-     *             password.
+     * @throws AuthenticationException Thrown if an invalid key is set
+     * @throws NoSuchAlgorithmException Thrown if the specified algorithm does not exist.
      * @throws RegistrationException
      *             Thrown if the user already exist in the system.
      */
     public static final void createUser(final String user, final String password, final String email, final boolean isInstructor)
-            throws GeneralSecurityException, RegistrationException {
+            throws AuthenticationException, NoSuchAlgorithmException, RegistrationException {
```

Comment:
Yes this was removed because the AuthenticationException extends a different class